specification/developersigning/DeveloperSigning/tspconfig.yaml

```
@@ -0,0 +1,32 @@
+parameters:
+  "service-dir":
+    default: "sdk/developer-signing"
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "examples"
```

This will avoid generating unnecessary schemas to your OpenAPI file:
```suggestion
    examples-directory: "examples"
    omit-unreachable-types: true
```

---

Great, commited to the branch